docs/external-tor-usage.md

```diff
@@ -0,0 +1,49 @@
+# Using External Tor with Haveno
+>Looking to setup external Tor for a seednode? Go [here](https://github.com/haveno-dex/haveno/blob/master/docs/deployment-guide.md#install-tor) for tailored instructions.
+## Setting up Tor:
+
+### 1. Download and Install Tor
+#### Linux
+You can install Tor through your system's package manager, or directly by going to torproject.org. Note that your package manager's version of tor might be outdated or non-GNU (see below note).
+#### MacOS
+You can install Tor through Homebrew (brew install tor), or directly by going to torproject.org. Note that the Homebrew version of tor might be outdated or non-GNU (see note below).
+#### Windows
+You can install Tor directly by going to torproject.org.
+> For best results, ensure that your version of tor matches the one currently used by Haveno. To improve loading times, it is highly recommended to use the GNU build, as it has support for POW. You can check if your build is GNU by running tor --version and looking for a response mentioning GNU.
+
+### 2. Add the following lines to your torrc:
+```
+controlPort=9051
+CookieAuthentication 1
+CookieAuthFile <path to cookie auth file>
+```
+> The "CookieAuthFile" path should be in a relatively stable and safe place. A good place to put it is in the same directory as torrc.
+
+### The location of your torrc file varies depending on your system:
```

Comment:
Should be '####' so it's not at same level as main numbers?

Reply:
These files do not exist by default after installing (at least on mac). Should they create it manually? Or start tor and stop tor to create the default file?